src/components/SignInSignUpModal/RegisterLoginModal/RegisterLoginModal.tsx

```diff
@@ -75,8 +74,12 @@ function RegisterLoginModal(): any {
           <IconButton aria-label="Close" className={styles.closeButton} onClick={handleClose}>
             <CloseIcon />
           </IconButton>
-          <RegisterLoginHeader checked={checked} onChange={handleChange} onClick={handleClose} state={state} />
-          {checked ? <RegisterForm changeStateFn={changeState} /> : <LoginForm changeStateFn={changeState} />}
+          <RegisterLoginHeader checked={checked} onChange={handleChange} onClick={handleClose} />
+          {checked ? (
+            <RegisterForm manageRequestMessage={changeState} />
+          ) : (
+            <LoginForm manageRequestMessage={() => changeState} />
```

Comment:
It's definitely OK but wouldn't `<LoginForm manageRequestMessage={changeState} />` work, just like above with `RegisterForm`?